posthog/api/insight.py

```diff
@@ -468,6 +468,11 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
             elif key == "my_last_viewed":
                 if str_to_bool(request.GET["my_last_viewed"]):
                     queryset = self._annotate_with_my_last_viewed_at(queryset).filter(my_last_viewed_at__isnull=False)
+            elif key == "feature_flag":
+                queryset = queryset.filter(
+                    Q(filters__breakdown__icontains=request.GET["feature_flag"])
```

Comment:
Quick flyby: API tests pleaase.

Also to prevent name clashes, probably check for insights containing this: (which represents the actual ff, and not just a property with the same name)

```suggestion
                    Q(filters__breakdown__icontains=f'$feature/{request.GET["feature_flag"]}')
```